python/mxnet/tvmop.py

```diff
@@ -31,7 +32,12 @@
     check_call(_LIB.MXLoadTVMOp(c_str(_LIB_TVM_OP[0])))
 
     # op sch config
-    _CONF_TVM_OP = find_conf_path("tvmop")
-    with open(_CONF_TVM_OP[0], "r") as f:
-        ret = ConfigSpaces.from_json_dict(json.load(f))
-    _set_tvm_op_config(ret)
+    try:
+        _CONF_TVM_OP = find_conf_path("tvmop")
+    except RuntimeError as e:
+        logging.warning(e)
```

Comment:
Could you add comments on what's the implication of this? Why we just print a warning. It's not very clear when going through the code.

Reply:
Comments added. The config file records available schedules, but besides these, a default schedule is also available, so when the conf is missing, we call fallback to default.

Reply:
Could you still add a message that explains this to the user? otherwise, they just see an error as warning without context.